packages/evm/contracts/court/AragonCourt.sol

```diff
@@ -86,15 +83,26 @@ contract AragonCourt is Controller, IArbitrator {
     */
     function createDispute(uint256 _possibleRulings, bytes calldata _metadata) external returns (uint256) {
         IArbitrable subject = IArbitrable(msg.sender);
-        require(subject.supportsInterface(ARBITRABLE_INTERFACE_ID), ERROR_SENDER_NOT_ARBITRABLE);
-
         ISubscriptions subscriptions = ISubscriptions(_getSubscriptions());
         require(subscriptions.isUpToDate(address(subject)), ERROR_SUBSCRIPTION_NOT_PAID);
 
         IDisputeManager disputeManager = IDisputeManager(_getDisputeManager());
         return disputeManager.createDispute(subject, _possibleRulings.toUint8(), _metadata);
     }
 
+    /**
+    * @notice Submit `_evidence` as evidence from `_submitter` for dispute #`_disputeId`
+    * @param _disputeId Id of the dispute in the Court
+    * @param _submitter Address of the account submitting the evidence
+    * @param _evidence Data submitted for the evidence related to the dispute
+    */
+    function submitEvidence(uint256 _disputeId, address _submitter, bytes calldata _evidence) external {
+        IDisputeManager disputeManager = IDisputeManager(_getDisputeManager());
+        (IArbitrable subject ,,,,,) = disputeManager.getDispute(_disputeId);
+        require(subject == IArbitrable(msg.sender), ERROR_SENDER_NOT_DISPUTE_SUBJECT);
```

Comment:
Yes, the main reason is to avoid relying on the Arbitrable to log the Arbitrator correctly. Additionally, we are now being flexible on whatever functionality users would like to provide for submitting evidence on their ends